06 - Type Ahead/index-START.html

```diff
@@ -15,8 +15,68 @@
     </ul>
```

Comment:
since this merge request has multiple files, maybe we could think about adding github pages. have you worked with this before?

Reply:
I deployed a couple of websites on GitHub pages but using a boilerplate that was doing everything for me through scripts. I would be happy to talk about it and learn a bit more

Reply:
resolving because we talked about it on the phone